src/app/(proper_react)/(redesign)/(public)/RebrandAnnouncement.module.scss

```diff
@@ -30,6 +30,11 @@
     background-color: transparent;
     border-style: none;
     cursor: pointer;
+    padding: 0 $spacing-xs;
```

Comment:
I'm not entirely sure, but I'm wondering if this means the `calc(2 * $spacing-sm)` on line 17 should be replaced by `calc(2 * $spacing-xs)`?

Reply:
In order to keep the previous spacing between the banner text content and the button: Yes. Updated in [885e790](https://github.com/mozilla/blurts-server/pull/4113/commits/885e79080cff5533df5511093622031f5a3f0a8d).